plans/tier1.fmf

```diff
@@ -133,3 +133,14 @@ discover+:
   - name: reboot machine
     how: ansible
     playbook: tests/ansible_collections/reboot.yml
+
+/yum_distro_sync:
+  discover+:
+    test: yum-distro-sync
```

Comment:
In the new test format for tier1 we use `checks-after-conversion` as a test itself , which is a collection of all checks that are run on the destination system (after conversion). Please add `yum-distro-sync` to the preparation phase and make sure the system is rebooted after this phase (after running your `yum-distro-sync`). I see that you put reboot after installing problematic packages. Unless it's necessary to reboot the machine after this part you can add the `yum-distro-sync` before the reboot.